web/html/src/manager/maintenance/calendar/web-calendar.tsx

```diff
@@ -0,0 +1,262 @@
+import * as React from "react";
+import { useState, useEffect, useRef } from "react";
+
+import FullCalendar from "@fullcalendar/react";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import timeGridPlugin from "@fullcalendar/timegrid";
+import interactionPlugin from "@fullcalendar/interaction";
+import allLocales from "@fullcalendar/core/locales-all";
+
+import { MessageType, Utils as MessagesUtils } from "components/messages";
+import Network from "utils/network";
+
+type WebCalendarProps = {
+  id: number;
+  type: string;
+  messages: (messages: MessageType[]) => void;
+  clearMessages: (messages: void) => void;
+  responseError: (messages: MessageType[]) => void;
+};
+
+type Event = {
+  start: string;
+  end: string;
+  title: string;
+}
+
+const WebCalendar = (props: WebCalendarProps) => {
+
+  const [events, setEvents] = useState<Event[]>([]);
+  const [currentDate, setCurrentDate] = useState<Date>();
+  const calendarRef = useRef<any>();
+
+  useEffect(() => {
+    getEvents("initial");
+    setCurrentDate(getApi().currentDataManager.data.currentDate);
+    // Hack: Make Fullcalendar button icons show correctly
+    jQuery(".fc-backButton-button").html('<div class="fa fa-angle-left"></div>');
```

Comment:
What's the problem that's being solved here? I understand that the default DOM somehow doesn't suit us, but what's the exact issue? Without being familiar with it, might one of these be of value here?  

 - https://fullcalendar.io/docs/buttonIcons  
 - https://fullcalendar.io/docs/themeSystem

Reply:
For some reason that I couldn't figure out adding the icons this way does not work here. They are only displayed as empty square (like it can't access the texture somehow)

Reply:
Fair enough. Imo it's not a showstopper to shipping this in any way, might be nice to create a separate ticket to follow up on this in the future.

Reply:
I added the issue to the card. So that we don't forget to look into it later on.